Gemfile

```diff
@@ -100,6 +100,7 @@ gem 'rails-html-sanitizer'
 # HTTP Clients
 # gem 'httparty' # Installed as a dependency via Avo
 # gem 'faraday' # Installed as a dependency via Cloudinary
+gem 'faraday-follow_redirects', '~> 0.3.0'
```

Comment:
Presumably does what it says on the tin - easiest way of achieving this?

Reply:
Yep, think this is easiest if we want to handle all requests with Faraday rather than HTTParty.